demo/api/src/products/entities/product-color.entity.ts

```diff
@@ -0,0 +1,44 @@
+import { CrudField, DocumentInterface } from "@comet/cms-api";
+import { BaseEntity, Entity, ManyToOne, OptionalProps, PrimaryKey, Property, Ref } from "@mikro-orm/core";
+import { Field, ID, ObjectType } from "@nestjs/graphql";
+import { v4 as uuid } from "uuid";
+
+import { Product } from "./product.entity";
+
+@ObjectType({
+    implements: () => [DocumentInterface],
+})
+@Entity()
+export class ProductColor extends BaseEntity<ProductColor, "id"> implements DocumentInterface {
+    [OptionalProps]?: "createdAt" | "updatedAt";
+
+    @PrimaryKey({ type: "uuid" })
+    @Field(() => ID)
+    id: string = uuid();
+
+    @Property()
+    @Field()
+    name: string;
+
+    @Property()
+    @Field()
+    colorHexCode: string;
```

Comment:
Nit: the color prefix isn't necessary IMO

Reply:
of course